up42/storage.py

```diff
@@ -259,13 +260,27 @@ def get_assets(
             or geometry is not None
             or custom_filter is not None
         ):
+            warn(
+                "Search for geometry or other STAC related fields will be deprecated soon."
```

Comment:
@andher1802 Do you know whether warnings support Markdown? In this case, we could include a link to https://sdk.up42.com/storage/#pystac

Reply:
Maybe we could align the text of this deprecation announcement with what we've discussed here: https://github.com/up42/up42-py/pull/497

Reply:
good idea, I was also thinking in having some common language as we are deprecating many things in the short-term.